Add 'clipboard' and 'trash' to the default features so we don't have
to maintain d/control manually
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,7 +42,7 @@
 
 [features]
 clipboard = ["terminal-clipboard"]
-default = []
+default = ["clipboard", "trash"]
 kitty-csi-check = ["xterm-query"]
 trash = ["dep:trash"]
 
